packages/kafka/kafka.spec

```diff
@@ -66,6 +66,7 @@ sed "s,log.dirs=.*,log.dirs=%{_sharedstatedir}/kafka," config/server.properties
 sed "s,dataDir=.*,dataDir=%{_sharedstatedir}/zookeeper," config/zookeeper.properties > %{buildroot}%{kafka_home}/config/zookeeper.properties
 cp -r config %{buildroot}%{kafka_home}/config-sample
 cp config/log4j.properties %{buildroot}%{kafka_home}/config
+cp config/tools-log4j.properties %{buildroot}%{kafka_home}/config
```

Comment:
Is there a different between the `log4j.properties` and `tools-log4j.properties`?